go.mod

```diff
@@ -1,5 +1,14 @@
 module github.com/wk8/go-ordered-map
```

Comment:
with v2, this must be `github.com/wk8/go-ordered-map/2` otherwise the go mod will be unhappy :)

Reply:
Nope, one should pinpoint whichever version they want to use in their go.mod instead. I'm really not a fan of the clunky way `/v2` got implemented.

Reply:
```shell
22-04-26 17:41 % go get github.com/wk8/go-ordered-map@v2
go: github.com/wk8/go-ordered-map@v2: no matching versions for query "v2"
22-04-26 18:05 % go get github.com/wk8/go-ordered-map@2 
go: github.com/wk8/go-ordered-map@2: invalid version: unknown revision 2
22-04-26 18:05 % go get github.com/wk8/go-ordered-map@v2.0.0
go: github.com/wk8/go-ordered-map@v2.0.0: invalid version: module contains a go.mod file, so module path must match major version ("github.com/wk8/go-ordered-map/v2")
22-04-26 18:05 % 
22-04-26 18:06 % go get github.com/wk8/go-ordered-map@latest
go: downloading github.com/wk8/go-ordered-map v1.0.0
go: added github.com/wk8/go-ordered-map v1.0.0
```

Reply:
Good catch, didn't know go mod enforced that now, that sucks :( fixing.

Reply:
Thanks @SVilgelm !

Reply:
thank you!
it works now:
```shell
22-04-26 18:27 % go get github.com/wk8/go-ordered-map/v2@latest
go: added github.com/bahlo/generic-list-go v0.2.0
go: added github.com/wk8/go-ordered-map/v2 v2.0.0
```

Reply:
actually, you are lucky that the go cache server did not memorize the wrong v2 release, otherwise you will need to release something like v2.0.1 with a fix and mark the v2 as excluded (https://go.dev/ref/mod#mvs-exclude)
🍀

Reply:
Indeed :)